doc/devices.rst

```diff
@@ -104,25 +104,142 @@ Each problem is unique, so it can often be best to choose the default behaviour
 
 **Multi-GPU/multi-node support:**
 
-The ``lightning.gpu`` device allows users to leverage the computational power of multi-node and multi-GPUs for running large-scale applications. To enable this feature, the following requirements need to be met:
-1. Both total number of `MPI` processes and `MPI` processes per node must be the same and a power of 2. For example, 2, 4, 8, 16, etc.. 
-2. Each MPI process is responsible for managing one GPU.
-Each `MPI` process is responsible for managing one GPU for the moment. 
-To enable this feature, users can set `mpi=True`, when configuring the lightning.gpu device. 
-
-Furthermore, users can fine-tune the performance of `MPI` operations by adjusting the `mpi_buf_size` parameter. This parameter determines the allocation of `mpi_buf_size` MB(megabytes) GPU memory buffer for `MPI` operations. 
-Note that there will be a runtime warning if GPU memory buffer for MPI operation is larger than the GPU memory allocated for the local state vector. Note `mpi_buf_size` should be also power of 2. Remember to carefully manage the mpi_buf_size parameter, taking into account 
-the available GPU memory and the memory requirements of the local state vector, to avoid memory issues and ensure optimal performance.
-By default (`mpi_buf_size=0`), the GPU memory allocated for MPI operations will be the same of size of the local state vector, with a limit of 64 MB(megabytes).
-
-The workflow for the new feature is:
+The ``lightning.gpu`` device allows users to leverage the computational power of multi-node and multi-GPUs for running large-scale simulations. 
+Provided that NVIDIA cuQuantum libraries, a ``CUDA-aware MPI`` library and ``mpi4py`` are properly installed and the path to the ``libmpi.so`` is 
+added to the ``LD_LIBRARY_PATH`` environment variable, the following requirements should be met to enable multi-node and multi-GPUs simulations:
+
+1. Both total number of MPI processes and MPI processes per node must be the same and a power of 2. For example, 2, 4, 8, 16, etc.. 
+2. Each MPI process is responsible for managing one GPU. Each MPI process is responsible for managing one GPU for the moment. 
```

Comment:
```suggestion
2. Each MPI process is responsible for managing one GPU.
```

Reply:
Done!

Reply:
Done, but no pushed yet, right?

Reply:
I think this could be fused with the point above.